agent/php_memcached.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef PHP_MEMCACHED_HDR
+#define PHP_MEMCACHED_HDR
+
+#include "nr_datastore_instance.h"
+#include "php_includes.h"
+
```

Comment:
Please add a doc string that will describe what does this function does.

Reply:
added in [a7bbbe9](https://github.com/newrelic/newrelic-php-agent/pull/958/commits/a7bbbe9f6788070cd01868488c69f3dba745ba3a)